README.md

```diff
@@ -13,9 +13,9 @@ Additional description about the project and its features.
 - Node.js
 - React
 
-## Live Demo
+<!-- ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://livedemo.com) -->
```

Comment:
- [ ] Kindly remove commented code. If you do not have the live link now, you can add it at another time.